src/holofuel/pages/CreateRequest/CreateRequest.js

```diff
@@ -66,7 +67,7 @@ export default function CreateRequest ({ history: { push } }) {
           {counterparty.length === AGENT_ID_LENGTH && <HashIcon hash={counterparty} size={26} />}
         </div>
         <div styleName='hash-nickname-wrapper'>
-          {counterparty.length === AGENT_ID_LENGTH && <h4 data-testid='counterparty-nickname'><RenderNickname agentId={counterparty} /></h4>}
+          {counterparty.length === AGENT_ID_LENGTH && <h4 data-testid='counterparty-nickname'><RenderNickname agentId={counterparty} setAgentNick={setCounterpartyNick} /></h4>}
```

Comment:
It would be more readable if the `set` function was called the same thing in both components. Either name works, I think I prefer `setAgentNick` but not strongly.